examples/zephyr/certificate_provisioning/README.md

```diff
@@ -0,0 +1,182 @@
+
+## Requirements
+
+* Golioth credentials
+* Network connectivity
+* A filesystem
+* ``mcumgr`` CLI tool
+
+## Building and Running
+
+### Platform specific configuration
+
+#### nRF52840 DK + ESP32-WROOM-32
+
+This subsection documents using nRF52840 DK running Zephyr with
+offloaded ESP-AT WiFi driver and ESP32-WROOM-32 module based board (such
+as ESP32 DevkitC rev. 4) running WiFi stack. See [AT Binary
+Lists](https://docs.espressif.com/projects/esp-at/en/latest/AT_Binary_Lists/index.html)
+for links to ESP-AT binaries and details on how to flash ESP-AT image on
+ESP chip. Flash ESP chip with following command:
+
+```console
+esptool.py write_flash --verify 0x0 PATH_TO_ESP_AT/factory/factory_WROOM-32.bin
+```
+
+Connect nRF52840 DK and ESP32-DevKitC V4 (or other ESP32-WROOM-32 based
+board) using wires:
+
+| nRF52840 DK | ESP32-WROOM-32  | ESP32-WROVER-32 |
+| ----------- | --------------- | ----------------|
+| P1.01 (RX)  | IO17 (TX)       | IO22 (TX)       |
+| P1.02 (TX)  | IO16 (RX)       | IO19 (RX)       |
+| P1.03 (CTS) | IO14 (RTS)      | IO14 (RTS)      |
+| P1.04 (RTS) | IO15 (CTS)      | IO15 (CTS)      |
+| P1.05       | EN              | EN              |
+| GND         | GND             | GND             |
+
+Configure the following Kconfig options based on your WiFi AP
+credentials:
+
+* GOLIOTH_SAMPLE_WIFI_SSID - WiFi SSID
+* GOLIOTH_SAMPLE_WIFI_PSK - WiFi PSK
+
+by adding these lines to configuration file (e.g. `prj.conf` or
+`board/nrf52840dk_nrf52840.conf`):
+
+```cfg
+CONFIG_GOLIOTH_SAMPLE_WIFI_SSID="my-wifi"
+CONFIG_GOLIOTH_SAMPLE_WIFI_PSK="my-psk"
+```
+
+On your host computer open a terminal window, locate the source code of
+this sample application (i.e., `examples/zephyr/certificate_provisioning`) and type:
+
+```console
+$ west build -b nrf52840dk_nrf52840 examples/zephyr/certificate_provisioning
+$ west flash
+```
+
+#### nRF9160 DK
+
+On your host computer open a terminal window, locate the source code of
+this sample application (i.e., `examples/zephyr/certificate_provisioning`) and type:
+
+```console
+$ west build -b nrf9160dk_nrf9160_ns examples/zephyr/certificate_provisioning
+$ west flash
+```
+
+## Installing `mcumgr`
+
+For full instructions, see
+[mcumgr](https://docs.zephyrproject.org/latest/services/device_mgmt/mcumgr.html).
+
+1. Install go from https://go.dev/doc/install
+2. Install the mcumgr tool:
+
+```console
+$ go install github.com/apache/mynewt-mcumgr-cli/mcumgr@latest
+```
+
+## Creating Certificates
+
+This sample requires that you have:
+
+* A root or intermediate certificate uploaded to the Golioth console
+* A client certificate signed by the private key associated with the
+  root or intermediate certificate
+* The private key associated with the client certificate
+
+For instructions on generating and positioning these certificates, see
+[golioth cert
+auth](https://docs.golioth.io/firmware/zephyr-device-sdk/authentication/certificate-auth).
```

Comment:
Probably want to change this to point to either the Firmware SDK docs or a generic section for generating certificates and adding them to the console.

Reply:
I think this link is relevant. It provides instructions for a procedure out-of-scope for the Firmware SDK just like the link to installing mcumgr. I could change the link to point to the scripts directory but there's little explanation there, and no discription of uploading the root cert to Golioth.

Reply:
I agree it's relevant, but right now it points to _Zephyr_ SDK documentation from the CP SDK, which is confusing.

Reply:
Ah, sorry. I didn't realize the issue was the wrong docs section.

We already have a cert provision

Reply:
Yup, that's good with me!